### PR TITLE
Add hbase logs

### PR DESCRIPTION
**Pull request by pducolin**
### What does this PR do?

Add hbase logs

### Motivation

Add support for hbase logs

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### CODE REVIEW

File: hbase_master/README.md [hunk 1]
```diff
@@ -47,15 +47,40 @@ If you are using Agent v6.8+ follow the instructions below to install the Hbase_
 
 2. [Restart the Agent][9]
 
+### Log collection
+
+_Available for Agent versions >6.0_
```
Comment (ChristineTChen) [inline hbase_master/README.md:52]:
```suggestion
```

We don't need this anymore 😄

File: hbase_master/README.md [hunk 2]
```diff
@@ -47,15 +47,40 @@ If you are using Agent v6.8+ follow the instructions below to install the Hbase_
...
+
+1. Collecting logs is disabled by default in the Datadog Agent, you need to enable it in `datadog.yaml`:
+
+   ```yaml
+   logs_enabled: true
+   ```
+
+2. Add this configuration block to your `hbase_master.d/conf.yaml` file to start collecting your Active Directory Logs:
```
Comment (ChristineTChen) [inline hbase_master/README.md:60]:
```suggestion
2. Add this configuration block to your `hbase_master.d/conf.yaml` file to start collecting your Hbase_master Logs:
```

File: hbase_master/README.md [hunk 3]
```diff
@@ -47,15 +47,40 @@ If you are using Agent v6.8+ follow the instructions below to install the Hbase_
...
+
+   ```yaml
+   logs:
+     - type: file
+       path: /path/to/my/directory/file.log
+       source: hbase
+       service: "<MY_SERVICE>"
```
Comment (ChristineTChen) [inline hbase_master/README.md:67]:
```suggestion
```

we don't need this here as we want customers to start using the service option in the init_config

File: hbase_master/README.md [hunk 4]
```diff
@@ -47,15 +47,40 @@ If you are using Agent v6.8+ follow the instructions below to install the Hbase_
...
+   ```
+
+   Change the `path` and `MY_SERVICE` parameter values and configure them for your environment.
```
Comment (ChristineTChen) [inline hbase_master/README.md:70]:
```suggestion
   Change the `path` parameter value and configure it for your environment.
```

File: hbase_master/README.md [hunk 5]
```diff
@@ -67,7 +92,7 @@ The Hbase_master check does not include any service checks.
 
 ## Troubleshooting
 
-Need help? Contact [Datadog support][12].
+Need help? Contact [Datadog support][14].
```
Comment (ChristineTChen) [inline hbase_master/README.md:95]:
```suggestion
Need help? Contact [Datadog support][12].
```

File: hbase_master/README.md [hunk 6]
```diff
@@ -47,15 +47,40 @@ If you are using Agent v6.8+ follow the instructions below to install the Hbase_
...
 
-[Run the Agent's `status` subcommand][10] and look for `hbase_master` under the Checks section.
+[Run the Agent's `status` subcommand][12] and look for `hbase_master` under the Checks section.
 
 ## Data Collected
 
 ### Metrics
 
-See [metadata.csv][11] for a list of metrics provided by this check.
+See [metadata.csv][13] for a list of metrics provided by this check.
```
Comment (ChristineTChen) [inline hbase_master/README.md:83]:
```suggestion
See [metadata.csv][11] for a list of metrics provided by this check.
```

File: hbase_master/README.md [hunk 7]
```diff
@@ -47,15 +47,40 @@ If you are using Agent v6.8+ follow the instructions below to install the Hbase_
...
+
+   Change the `path` and `MY_SERVICE` parameter values and configure them for your environment.
+   See the [sample hbase_master.d/conf.yaml][10] for all available configuration options.
+
+3. [Restart the Agent][11].
+
 ### Validation
 
-[Run the Agent's `status` subcommand][10] and look for `hbase_master` under the Checks section.
+[Run the Agent's `status` subcommand][12] and look for `hbase_master` under the Checks section.
```
Comment (ChristineTChen) [inline hbase_master/README.md:77]:
```suggestion
[Run the Agent's `status` subcommand][10] and look for `hbase_master` under the Checks section.
```

File: hbase_master/README.md [hunk 8]
```diff
@@ -47,15 +47,40 @@ If you are using Agent v6.8+ follow the instructions below to install the Hbase_
...
+     - type: file
+       path: /path/to/my/directory/file.log
+       source: hbase
+       service: "<MY_SERVICE>"
+   ```
+
+   Change the `path` and `MY_SERVICE` parameter values and configure them for your environment.
+   See the [sample hbase_master.d/conf.yaml][10] for all available configuration options.
+
+3. [Restart the Agent][11].
```
Comment (ChristineTChen) [inline hbase_master/README.md:73]:
```suggestion
3. [Restart the Agent][9].
```

File: hbase_master/datadog_checks/hbase_master/data/conf.yaml.example [hunk 9]
```diff
@@ -171,3 +171,30 @@ init_config:
...
+## service - optional - The name of the service that generates the log.
+##                      Overrides any `service` defined in the `init_config` section.
+## tags - optional - Add tags to the collected logs.
+##
+## Discover Datadog log collection: https://docs.datadoghq.com/logs/log_collection/
+#
+# logs:
+#   - type: file
+#     path: /var/log/hbase/hbase-hbase-master-*.log
+#     service: hbase
```
Comment (ChristineTChen) [inline hbase_master/datadog_checks/hbase_master/data/conf.yaml.example:191]:
```suggestion
```

File: hbase_master/datadog_checks/hbase_master/data/conf.yaml.example [hunk 10]
```diff
@@ -171,3 +171,30 @@ init_config:
...
+#     source: hbase
+#   - type: file
+#     path: /var/log/hbase/hbase-hbase-rest-*.log
+#     service: hbase
```
Comment (ChristineTChen) [inline hbase_master/datadog_checks/hbase_master/data/conf.yaml.example:195]:
```suggestion
```

File: hbase_master/datadog_checks/hbase_master/data/conf.yaml.example [hunk 11]
```diff
@@ -171,3 +171,30 @@ init_config:
...
+#     source: hbase
+#   - type: file
+#     path: /var/log/hbase/hbase-hbase-thrift-*.log
+#     service: hbase
```
Comment (ChristineTChen) [inline hbase_master/datadog_checks/hbase_master/data/conf.yaml.example:199]:
```suggestion
```

File: hbase_regionserver/README.md [hunk 12]
```diff
@@ -49,15 +49,40 @@ If you are using Agent v6.8+ follow the instructions below to install the HBase
 
 2. [Restart the Agent][10]
 
+### Log collection
+
+_Available for Agent versions >6.0_
```
Comment (ChristineTChen) [inline hbase_regionserver/README.md:54]:
```suggestion
```

File: hbase_regionserver/README.md [hunk 13]
```diff
@@ -49,15 +49,40 @@ If you are using Agent v6.8+ follow the instructions below to install the HBase
...
+
+1. Collecting logs is disabled by default in the Datadog Agent, you need to enable it in `datadog.yaml`:
+
+   ```yaml
+   logs_enabled: true
+   ```
+
+2. Add this configuration block to your `hbase_regionserver.d/conf.yaml` file to start collecting your Active Directory Logs:
```
Comment (ChristineTChen) [inline hbase_regionserver/README.md:62]:
```suggestion
2. Add this configuration block to your `hbase_regionserver.d/conf.yaml` file to start collecting your Hbase_regionserver Logs:
```

File: hbase_regionserver/README.md [hunk 14]
```diff
@@ -49,15 +49,40 @@ If you are using Agent v6.8+ follow the instructions below to install the HBase
...
+
+   ```yaml
+   logs:
+     - type: file
+       path: /path/to/my/directory/file.log
+       source: hbase
+       service: "<MY_SERVICE>"
```
Comment (ChristineTChen) [inline hbase_regionserver/README.md:69]:
```suggestion
```

File: hbase_regionserver/README.md [hunk 15]
```diff
@@ -49,15 +49,40 @@ If you are using Agent v6.8+ follow the instructions below to install the HBase
...
+   ```
+
+   Change the `path` and `MY_SERVICE` parameter values and configure them for your environment.
```
Comment (ChristineTChen) [inline hbase_regionserver/README.md:72]:
```suggestion
   Change the `path` parameter value and configure it for your environment.
```

File: hbase_regionserver/README.md [hunk 16]
```diff
@@ -49,15 +49,40 @@ If you are using Agent v6.8+ follow the instructions below to install the HBase
...
+   See the [sample hbase_regionserver.d/conf.yaml][11] for all available configuration options.
+
+3. [Restart the Agent][12].
```
Comment (ChristineTChen) [inline hbase_regionserver/README.md:75]:
```suggestion
3. [Restart the Agent][10].
```

File: hbase_regionserver/README.md [hunk 17]
```diff
@@ -49,15 +49,40 @@ If you are using Agent v6.8+ follow the instructions below to install the HBase
...
+   ```yaml
+   logs:
+     - type: file
+       path: /path/to/my/directory/file.log
+       source: hbase
+       service: "<MY_SERVICE>"
+   ```
+
+   Change the `path` and `MY_SERVICE` parameter values and configure them for your environment.
+   See the [sample hbase_regionserver.d/conf.yaml][11] for all available configuration options.
```
Comment (ChristineTChen) [inline hbase_regionserver/README.md:73]:
```suggestion
   See the [sample hbase_regionserver.d/conf.yaml][9] for all available configuration options.
```

File: hbase_regionserver/README.md [hunk 18]
```diff
@@ -49,15 +49,40 @@ If you are using Agent v6.8+ follow the instructions below to install the HBase
...
+
+3. [Restart the Agent][12].
+
 ## Validation
 
-[Run the Agent's `status` subcommand][11] and look for `hbase_regionserver` under the Checks section.
+[Run the Agent's `status` subcommand][13] and look for `hbase_regionserver` under the Checks section.
```
Comment (ChristineTChen) [inline hbase_regionserver/README.md:79]:
```suggestion
[Run the Agent's `status` subcommand][11] and look for `hbase_regionserver` under the Checks section.
```

File: hbase_regionserver/README.md [hunk 19]
```diff
@@ -49,15 +49,40 @@ If you are using Agent v6.8+ follow the instructions below to install the HBase
...
 
 ## Data Collected
 
 ### Metrics
 
-See [metadata.csv][12] for a list of metrics provided by this check.
+See [metadata.csv][14] for a list of metrics provided by this check.
```
Comment (ChristineTChen) [inline hbase_regionserver/README.md:85]:
```suggestion
See [metadata.csv][12] for a list of metrics provided by this check.
```

File: hbase_regionserver/README.md [hunk 20]
```diff
@@ -69,7 +94,7 @@ The HBase RegionServer check does not include any service checks.
 
 ## Troubleshooting
 
-Need help? Contact [Datadog support][13].
+Need help? Contact [Datadog support][15].
```
Comment (ChristineTChen) [inline hbase_regionserver/README.md:97]:
```suggestion
Need help? Contact [Datadog support][13].
```

File: hbase_regionserver/assets/configuration/spec.yaml [hunk 21]
```diff
@@ -1,13 +1,19 @@
...
+              host.value.example: localhost
+              port.value.example: 10102
+      - template: init_config
+        options:
+          - template: init_config/jmx
+      - template: logs
+        example:
+          - type: file
+            path: "/var/log/hbase/hbase-hbase-regionserver-*.log"
+            service: "hbase"
```
Comment (ChristineTChen) [inline hbase_regionserver/assets/configuration/spec.yaml:18]:
```suggestion
```

**Review by ChristineTChen — CHANGES_REQUESTED** [review]
Looks good! Left some comments to fix the links as they're pointing to wrong/non-existent reference numbers and remove some options that are considered outdated, e.g `service` field

File: hbase_master/README.md (hunk 5 above)
Comment (pducolin) [reply hbase_master/README.md:95]: And this is how I finally understood how reference-style links work

**Review by hithwen — APPROVED** [review]

**Review by ChristineTChen — APPROVED** [review]

**Pull request merged**